### DISCUSSION TITLE
window系统如何开启终端代理

### DISCUSSION THREAD

**Discussion by ZIA-Hans** [clash-verge-rev/clash-verge-rev#1797] [Q&A]
ClashX Pro软件在mac上有一个"增强模式"可以让终端上跑的一些请求命令也走vpn，在这个软件上如何设置这样的效果呢，是全局模式吗？求问

**Comment by wonfen** [comment]
所谓的“增强模式”，就是tun(虚拟网卡)模式。

**Comment by kty00** [comment]
走环境变量，tun 我都不敢用，怕有问题。什么增强模式，clashx pro 瞎取名字，太模糊，还是本意好一些，望文生义，就tun。
